packages/appstore/src/components/modals/real-wallets-upgrade/components/wallet-linking-step/wallet-linking-step.tsx

```diff
@@ -51,8 +55,52 @@ const WalletLinkingStep = observer(({ data }: TWalletLinkingStep) => {
                     return (
                         <WalletLinkWrapper
                             key={wallet.wallet_details.name}
-                            account_list={wallet.account_list}
-                            wallet_details={wallet.wallet_details}
+                            has_left_fork={wallet.account_list.length > 1 || is_mobile}
+                            left={() => {
+                                return (
+                                    <>
+                                        {is_mobile && (
+                                            <Text
+                                                as='div'
+                                                className='wallet-linking-step__title-text wallet-linking-step__accounts-text '
+                                                color='prominent'
+                                                size='xxxs'
+                                            >
+                                                <Localize i18n_default_text='Your current trading account(s)' />
+                                            </Text>
+                                        )}
+                                        {wallet.account_list.map(account => {
+                                            return (
+                                                <WalletAccount
+                                                    key={`${account.account_name}-${account.currency}}`}
+                                                    balance={account.balance}
+                                                    currency={account.currency}
+                                                    icon={account.icon}
+                                                    name={account.account_name}
+                                                    is_mobile={is_mobile}
+                                                />
+                                            );
+                                        })}
+                                    </>
+                                );
+                            }}
+                            center={() => <Icon icon='IcAppstoreWalletsLink' size={40} />}
+                            right={() => {
+                                return (
+                                    <div style={{ height: '13.6rem', position: 'relative' }}>
```

Comment:
Wouldn't it be better to add classname to this div with the css properties instead of inline styling?